src/ExperimentTable.js

```diff
@@ -0,0 +1,174 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Table } from 'evergreen-ui'
+import _ from 'lodash'
+
+import TableFooter from './TableFooter'
+import tableHeaderCells from './tableHeaderCells'
+
+class ExperimentTable extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      searchQuery: ``,
+      orderedColumn: 0,
+      searchedColumn: 1,
+      ordering: true,
+      checkedArray: [],
+      currentPage: 1,
+      entryPerPage: 10
+    }
+
+    this.sort = this.sort.bind(this)
+    this.filter = this.filter.bind(this)
+    this.setValue = this.setValue.bind(this)
+    this.handleCheckbox = this.handleCheckbox.bind(this)
+  }
+
+  sort(data) {
+    const {ordering, orderedColumn} = this.state
+    const propKey = this.props.tableHeader[orderedColumn].dataParam
+    const filteredElements = _.sortBy(data, propKey)
+    return ordering ? filteredElements :  filteredElements.reverse()
+  }
+
+  filter(data, tableHeader) {
+    const searchQuery = this.state.searchQuery.trim()
+    return searchQuery.length === 0 ? data :
+      data.filter(profile => _.isArray(profile[tableHeader[this.state.searchedColumn].dataParam]) ?
+        _.flattenDeep(profile[tableHeader[this.state.searchedColumn].dataParam]).some(item=>item.toLowerCase().includes(searchQuery.toLowerCase())) :
+        profile[tableHeader[this.state.searchedColumn].dataParam].toString().toLowerCase().includes(searchQuery.toLowerCase())
+      )
+  }
+
+  setValue(property){
+    return e => {
+      this.setState({
+        [property]: e.target.value,
+        currentPage: 1
+      })
+    }
+  }
+
+  handleCheckbox(accession) {
+    const checkedArray = this.state.checkedArray.slice()
```

Comment:
This behaviour of `slice` is a lucky side effect. Use [`Array.from()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from), since that expresses what you want to do.